Assets/Scripts/InputListener.cs

```diff
@@ -22,6 +22,7 @@ public class InputListener : MonoBehaviour {
         if (Input.GetKeyDown(KeyCode.R)) {
             Scene scene = SceneManager.GetActiveScene();
             SceneManager.LoadScene(scene.name);
+            Tutorial.GetSingleton().ResetTutorial();
         }
 
         if (Input.GetKeyDown(KeyCode.T)) {
```

Comment:
The T button does restart with tutorial. But maybe remove it since R and T do the same thing now